client/src/pages/Profile.js

```diff
@@ -1,24 +1,153 @@
+  const [name, setName] = useState("Steve Jobs");
+  const [about, setAbout] = useState("Apple Chief Officer");
+  const [files, setFiles] = useState([]);
+  const skills = [
+    { language: "JavaScript", level: "Advanced" },
+    { language: "Java", level: "Expert" },
+    { language: "PHP", level: "Expert" },
+    { language: "C++", level: "Beginner" },
+  ];
+  const projects = [
+    { title: "JS Code Review", link: "http://localhost:3000/" },
+    { title: "New Java Code", link: "http://localhost:3000/" },
+  ];
+
+  const handleEdit = () => {
+    setEdit(!edit);
+  };
+
+  const submit = (event) => {
+    event.preventDefault();
+    console.log("Submit!");
+    handleEdit();
+  };
+  const editName = (event) => setName(event.target.value);
+
+  const editAbout = (event) => setAbout(event.target.value);
+
   return (
     <Background solid>
-      <Grid container direction="column">
-        <Grid item>
-          <Navbar state={state} dispatch={dispatch} />
-        </Grid>
-        <Grid item container>
-          <Typography component="h1" variant="h6">
-            Profile
-          </Typography>
+      <Box flexWrap="nowrap" width={"100%"}>
+        <Grid container direction="column">
+          <Grid item>
+            <Navbar state={state} dispatch={dispatch} />
+          </Grid>
+          <Grid item>
+            <Container component="main" maxWidth="md" className={classes.root}>
+              <Avatar
+                alt="Profile image"
+                src={AvatarImage}
+                className={classes.avatar}
+              />
+              <Paper className={classes.paper}>
+                <Grid
+                  item
+                  container
+                  direction="column"
+                  justify="center"
+                  alignItems="center"
+                >
+                  <ProfileName
+                    edit={edit}
+                    name={name}
+                    about={about}
+                    editName={editName}
+                    editAbout={editAbout}
+                    files={files}
+                    setFiles={setFiles}
+                  />
+
+                  <Grid item xs={1}>
+                    {!edit && (
+                      <EditIcon
+                        fontSize="large"
+                        className={classes.iconEdit}
+                        onClick={handleEdit}
+                      />
+                    )}
+                    {edit && (
+                      <Button
+                        variant="contained"
+                        size="small"
+                        className={classes.button}
+                        onClick={submit}
+                      >
+                        Save
+                      </Button>
+                    )}
+                  </Grid>
+                  <Grid item>
+                    <Box mt={8} />
+                  </Grid>
+                  <ProfileStats years="5" reviews="10" rating="4.7" />
```

Comment:
this could also be fetched from user. number of reviews may need to be fetched separately unless it is saved in user.

Reply:
Yes, this should be done once we have reviews as a thing in the back-end

Reply:
For now I'll leave it hard coded and once we'll have that info, will pass appropriate values